sssoja/tv-bland
# applies automated lint tools

### What changes have you made?
- Set up automated lint tools combining ESLint with Prettier to organise imports and carry out automated syntax scans
- Configured the `.eslintrcjs` file to depend on the `@typescript-eslint/parser` and extend the plug-in `react/recommended`, which contains React-specific linting rules
- Removed all instances of importing `React` as Next.js does not require you to import React into each component

### Which issue(s) does this PR fix?
Fixes #16 

### Screenshots (if there are design changes)
No design changes

### How to test
Move through the codebase and check that imports are following the `sort-import` rule and the ESLint rules as configured in `.eslintrcjs` are applied throughout
